docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| phar             |     |     |     | *   | *   | *   | *   | *   | *   |
+| pinba            | *   | *   | *   |     |     |     |     |     |     |
+| posix            |     |     |     | *   | *   | *   | *   | *   | *   |
+| propro           |     |     | *   |     |     |     |     |     |     |
+| pspell           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pthreads         |     |     |     |     | *   |     |     |     |     |
+| raphf            |     |     | *   |     |     |     |     | *   | *   |
+| readline         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| recode           | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| redis            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| shmop            |     |     |     | *   | *   | *   | *   | *   | *   |
+| simplexml        |     |     |     | *   | *   | *   | *   | *   | *   |
+| snmp             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| soap             |     |     |     | *   | *   | *   | *   | *   | *   |
+| sockets          |     |     |     | *   | *   | *   | *   | *   | *   |
+| sodium           |     |     |     |     |     | *   | *   | *   | *   |
+| sourceguardian   |     |     |     | *   | *   |     |     |     |     |
+| spplus           | *   | *   |     |     |     |     |     |     |     |
+| sqlite3          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| sqlsrv           |     |     |     | *   | *   | *   |     | *   |     |
+| ssh2             | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| sybase           |     |     |     |     | *   | *   | *   | *   | *   |
+| sysvmsg          |     |     |     | *   | *   | *   | *   | *   | *   |
+| sysvsem          |     |     |     | *   | *   | *   | *   | *   | *   |
+| sysvshm          |     |     |     | *   | *   | *   | *   | *   | *   |
+| tideways         |     |     |     | *   | *   | *   | *   | *   | *   |
+| tideways-xhprof  |     |     |     | *   | *   | *   | *   | *   | *   |
+| tidy             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| tokenizer        |     |     |     | *   | *   | *   | *   | *   | *   |
+| uuid             |     |     |     |     | *   | *   | *   | *   | *   |
+| wddx             |     |     |     | *   | *   | *   | *   | *   |     |
+| xcache           | *   | *   |     |     |     |     |     |     |     |
+| xhprof           | *   | *   | *   |     |     |     |     |     |     |
+| xml              |     |     |     | *   | *   | *   | *   | *   | *   |
+| xmlreader        |     |     |     | *   | *   | *   | *   | *   | *   |
+| xmlrpc           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| xmlwriter        |     |     |     | *   | *   | *   | *   | *   | *   |
+| xsl              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| yaml             |     |     |     |     | *   | *   | *   | *   |     |
```

Comment:
This is 3rd party, yes?

Reply:
Yes 3rd party but ready soonTM

Reply:
I've updated the image and the doc will be included on the next php8 image release